packages/server/src/script/scanForFollowUps.ts

```diff
@@ -0,0 +1,76 @@
+// The functionality implemented in this file allows us to scan sign ups
```

Comment:
Can we follow the model of resendEmails instead? It seems like this generates a file to followups to be sent, then the other script actually sends them.  Is there a reason that you opted for this design? (e.g. sometimes there are sometimes issues with the output file from scanning that need to be examined by a human)

Reply:
Actually, we don't store a list of follow ups to be sent, we just make sure the collection in Firestore will have a field that allows us to know which voters have received their follow up email (we can't query for `undefined` or non-existant fields).

The good side of this approach is that if there's any execution error while you're running this we we'll not send any duplicates since we save the voters who have already received their email.

Happy to change how this works if you think this is not the best approach here.

Reply:
Ah, I understand, so this is a backfill?  Ok please rename this file "backfillFollowUpField.ts" or something then.  Otherwise, LGTM.